src/pam_hbac_ldap.c

```diff
@@ -509,6 +508,11 @@ static int secure_connection(pam_handle_t *ph,
                              const char *ca_cert,
                              bool secure)
 {
+#if defined(HPUX)
+#undef HAVE_LDAP_START_TLS                      # couldn't get it to work on hpux
```

Comment:
Here I wonder if it would be better to add a configure-time flag, something like `--disable-ssl` that could be used on HP-UX. Or even don't check for these functions at all during configure time to kind of pretend they don't exist.

By the way, can you share the error mesages from pam_hbac that you were seeing on HP-UX? Maybe we can figure out the solution..